app/mainAppWindow/index.js

```diff
@@ -168,10 +169,19 @@ function createWindow() {
 			nativeWindowOpen: true,
 			plugins: true,
 			nodeIntegration: false,
+      contextIsolation: false,
```

Comment:
This seems to be a bit to the "left". I guess either using tabs or spaces but it whould be better if it is at the same level as nodeIntegration: false and others. It could just be the github reviewing tool

Reply:
My default setting was to spaces instead of tabs. Which i changed now in editor. Also fixed all the ESLint issues generated with my code changes. I've pushed them already.